timur/lib/client/jsx/components/search/search.jsx

```diff
@@ -37,23 +37,9 @@ import {
 import ModelViewer from '../model_viewer';
 import useAsyncWork from "etna-js/hooks/useAsyncWork";
 import SearchQuery from "./search_query";
-import {Loading} from "etna-js/components/Loading";
```

Comment:
My editor was showing this stuff isn't actually used anymore (the other loader supercedes it).